ruby3.2-llhttp.yaml

```
@@ -0,0 +1,157 @@
+    LLHTTP_FFI_VERSION: 0.5.0
+    LLHTTP_MRI_VERSION: 0.6.0
+
+# The original release tag is in the format: 2023-03-29. We need to convert
+# back into that format before performing a git clone.
+var-transforms:
+  - from: ${{package.version}}
+    match: \.
+    replace: "-"
+    to: mangled-package-version
+
+vars:
+  gem-ffi: llhttp-ffi
+  gem-mri: llhttp
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: eb61456a325d87fcbc9e4c5a7512d9589737abae
+      repository: https://github.com/bryanp/llhttp.git
+      tag: ${{vars.mangled-package-version}}
+
+  - runs: |
+      bundle install && bundle update
+      gem pristine debug
+      gem pristine rbs
+
+subpackages:
+  - name: "${{package.name}}-mri"
+    description: "Ruby bindings for llhttp."
+    pipeline:
+      - working-directory: mri
+        pipeline:
+          - runs: |
+              bundle install && bundle update
+              rake compile
+              rake test
+          - uses: ruby/build
+            with:
+              gem: ${{vars.gem-mri}}
+          - name: Sanity check, compare ACTUAL_LLHTTP_VERSION with LLHTTP_VERSION
+            runs: |
+              export ACTUAL_LLHTTP_VERSION=$(grep -oP 'VERSION = "\K[0-9]+\.[0-9]+\.[0-9]+' lib/llhttp/version.rb)
+              if [ "${ACTUAL_LLHTTP_VERSION}" != "${LLHTTP_MRI_VERSION}" ]; then
+                echo "Expected version: ${LLHTTP_MRI_VERSION}, but got: ${ACTUAL_LLHTTP_VERSION}. Please update the version in the environment section."
+                exit 1
+              fi
+              mv ${{vars.gem-mri}}-*.gem ${{vars.gem-mri}}.gem
+          - uses: ruby/install
+            with:
+              gem-file: ${{vars.gem-mri}}.gem
+              version: ${LLHTTP_MRI_VERSION}
+          - uses: ruby/clean
+
+  - name: "${{package.name}}-ffi"
+    description: "Ruby FFI bindings for llhttp."
+    dependencies:
+      runtime:
+        - ruby3.2-ffi
+    pipeline:
+      - working-directory: ffi
+        pipeline:
+          - runs: |
+              bundle install && bundle update
+              rake compile
+              rake test
+          - uses: ruby/build
+            with:
+              gem: ${{vars.gem-ffi}}
+          - name: Sanity check, compare ACTUAL_LLHTTP_VERSION with LLHTTP_VERSION
+            runs: |
+              export ACTUAL_LLHTTP_VERSION=$(grep -oP 'VERSION = "\K[0-9]+\.[0-9]+\.[0-9]+' lib/llhttp/version.rb)
+              if [ "${ACTUAL_LLHTTP_VERSION}" != "${LLHTTP_FFI_VERSION}" ]; then
+                echo "Expected version: ${LLHTTP_FFI_VERSION}, but got: ${ACTUAL_LLHTTP_VERSION}. Please update the version in the environment section."
+                exit 1
+              fi
+              mv ${{vars.gem-ffi}}-*.gem ${{vars.gem-ffi}}.gem
+          - uses: ruby/install
+            with:
+              gem-file: ${{vars.gem-ffi}}.gem
+              version: ${LLHTTP_FFI_VERSION}
+          - uses: ruby/clean
+
+update:
+  enabled: true
```

Explicitly adding 'manual:true' here means the automation won't update and auto-merge the changes here. I'd it'll go onto the sustaining list for package updates and approvals

```suggestion
  enabled: true
  manual: true
```